syntax = "proto3";

package analyze_service;

// YksittÃ¤isen sarjan analyysi
message AnalyzeRequest {
  string series_id = 1;
}

// Kaikkien sarjojen analyysi
message AnalyzeAllRequest {}

// Vastaus analyysille
message AnalyzeResponse {
  string message = 1;
  string series_id = 2;
}

// gRPC-palvelun mÃ¤Ã¤rittely
service AnalyzeService {
  rpc AnalyzeDicomData (AnalyzeRequest) returns (AnalyzeResponse);
  rpc AnalyzeAllDicomData (AnalyzeAllRequest) returns (AnalyzeResponse);  // ðŸ”¹ UUSI METODI
}

